generators/templates/package.json

```diff
@@ -65,5 +65,8 @@
     "wct-browser-legacy": "^1.0.1",
     "web-component-tester": "^6.7.1"
   },<% } %>
-  "generator-pfelement-version": "<%= generatorPfelementVersion %>"
+  "generator-pfelement-version": "<%= generatorPfelementVersion %>",
+  "bugs": {
+    "url" : "https://github.com/patternfly/patternfly-elements/issues"
```

Comment:
@eyevana What would you think of incorporating @starryeyez024's idea of adding `?q=is%3Aissue+is%3Aopen+<%= elementName %>` to the url?

Reply:
Definitely. I think this a great idea. I made a note to go back and update them in the pfe project as well.